components/suggestion/suggestion_box.jsx

```diff
@@ -498,8 +498,8 @@ export default class SuggestionBox extends React.Component {
 
             if (handled) {
                 this.setState({
-                    presentationType: provider.presentationType(),
-                    allowDividers: provider.allowDividers(),
+                    presentationType: provider.presentationType,
+                    allowDividers: provider.allowDividers,
```

Comment:
Not sure about this fix.  `EmoticonProvider` don't have implementation of `presentationType` and `allowDividers`, so I guess we'll have to add those there.

Reply:
will be `undefined`, I'm not sure as well. I can add the implementation in the missing ones like the commandprovider and emoiticonprovider, but i will need to know which values to add.

Reply:
With the way we're currently showing it, I'm guessing it's like:
- EmoticonProvider - "allowDividers: false" and "presentationType: text"
- CommandProvider - "allowDividers: true" and "presentationType: text"

Reply:
will update, thanks so much for the review @saturninoabril